lib/Generate.js

```diff
@@ -44,7 +44,7 @@ function prepareCompiledJsFile(
 // variant of the `Test` type. To avoid having to update this regex if a new
 // variant is added, newer versions of elm-explorations/test have prefixed all
 // variants with `ElmTestVariant__` so we can match just on that.
-const testVariantDefinition = /^var\s+\$elm_explorations\$test\$Test\$Internal\$(ElmTestVariant__\w+|UnitTest|FuzzTest|Labeled|Skipped|Only|Batch)\s*=\s*(?:\w+\(\s*)?function\s*\([\w, ]*\)\s*\{\s*return *\{\s*\$:\s*(['"])\1\2/gm;
+const testVariantDefinition = /^var\s+\$elm_explorations\$test\$Test\$Internal\$(?:ElmTestVariant__\w+|UnitTest|FuzzTest|Labeled|Skipped|Only|Batch)\s*=\s*(?:\w+\(\s*)?function\s*\([\w, ]*\)\s*\{\s*return *\{/gm;
```

Comment:
What does `(?:\w+\(\s*)?` do?

Reply:
It matches the `F2` wrapper that the `Labeled` variant has:

```js
var $elm_explorations$test$Test$Internal$Labeled = F2(
	function (a, b) {
		return {$: 'Labeled', a: a, b: b};
	});
```

I tried to future-proof by not caring about the exact name.

Reply:
ah yes! Of course. cool